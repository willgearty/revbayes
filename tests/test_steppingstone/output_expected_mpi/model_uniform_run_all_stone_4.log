Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-21599.5	-21593.67	-5.821275	0.8697314	0.10056	0.0613729	0.149688	0.0837206	0.15303	0.168455	0.147763	0.11746	0.128203	0.209568	0.228521	0.0821517	0.104163	0.204816	0.161118	0.141454	0.130392	0.165411	0.0903366	0.253078	0.251773	0.273178	0.30352	0.150942	0.0987882	0.00116972	0.135251	0.145617	0.0424455	0.303692	0.0743351	0.0665388	0.0875717	0.00808999	0.054436	0.159162	0.00950557	0.0191636	0.13191	0.0106279	0.140913	0.0930225	0.160833	0.0175937	0.238062	0.0887537	0.0394932	0.576419	0.0396781	0.110287	0.433764	0.97455	2.4814	0.368957	0.297721	0.0946341	0.238688	0.4539272	5.603745
10	-21554.08	-21553.6	-0.48042	0.9762057	0.0557986	0.156626	0.121806	0.122795	0.0958712	0.310847	0.204916	0.062634	0.0523174	0.204318	0.205821	0.139268	0.0883798	0.206292	0.191772	0.0837358	0.153482	0.180867	0.179995	0.166654	0.103991	0.16084	0.217643	0.129689	0.0664503	0.00209908	0.0254001	0.101114	0.0485368	0.27632	0.109719	0.0973768	0.0601807	0.00526257	0.0882511	0.0840392	0.0534953	0.0136643	0.080094	0.00777636	0.0730386	0.0959169	0.198958	0.0199604	0.2432	0.0833278	0.0257823	0.596316	0.0314142	0.132182	0.469481	0.995943	2.40239	0.388528	0.315529	0.100295	0.195648	0.4104178	5.08405
