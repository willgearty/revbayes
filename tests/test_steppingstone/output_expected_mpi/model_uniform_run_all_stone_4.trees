Iteration	Posterior	Likelihood	Prior	psi
0	-21599.5	-21593.67	-5.821275	(Galeopterus_variegatus[&index=23]:0.303520,Tarsius_syrichta[&index=21]:0.251773,((Daubentonia_madagascariensis[&index=7]:0.147763,((Varecia_variegata_variegata[&index=22]:0.273178,(Lemur_catta[&index=10]:0.209568,(((Microcebus_murinus[&index=14]:0.204816,Cheirogaleus_major[&index=4]:0.083721)[&index=39]:0.131910,Propithecus_coquereli[&index=19]:0.090337)[&index=35]:0.054436,Lepilemur_hubbardorum[&index=11]:0.228521)[&index=38]:0.019164)[&index=40]:0.010628)[&index=27]:0.135251,(Loris_tardigradus[&index=12]:0.082152,(((Otolemur_crassicaudatus[&index=16]:0.141454,Galago_senegalensis[&index=8]:0.117460)[&index=24]:0.150942,Perodicticus_potto[&index=18]:0.165411)[&index=29]:0.042445,Nycticebus_coucang[&index=15]:0.161118)[&index=37]:0.009506)[&index=41]:0.140913)[&index=28]:0.145617)[&index=36]:0.159162,((((Cebus_albifrons[&index=3]:0.149688,Saimiri_sciureus[&index=20]:0.253078)[&index=26]:0.001170,Aotus_trivirgatus[&index=1]:0.100560)[&index=33]:0.087572,Callicebus_donacophilus[&index=2]:0.061373)[&index=43]:0.160833,((Hylobates_lar[&index=9]:0.128203,Pan_paniscus[&index=17]:0.130392)[&index=32]:0.066539,(Chlorocebus_aethiops[&index=5]:0.153030,(Macaca_mulatta[&index=13]:0.104163,Colobus_guereza[&index=6]:0.168455)[&index=25]:0.098788)[&index=42]:0.093022)[&index=30]:0.303692)[&index=31]:0.074335)[&index=34]:0.008090)[&index=44];
10	-21554.08	-21553.6	-0.48042	(Tarsius_syrichta[&index=21]:0.103991,(((Hylobates_lar[&index=9]:0.052317,(Pan_paniscus[&index=17]:0.153482,((Colobus_guereza[&index=6]:0.310847,Macaca_mulatta[&index=13]:0.088380)[&index=25]:0.066450,Chlorocebus_aethiops[&index=5]:0.095871)[&index=42]:0.095917)[&index=32]:0.097377)[&index=30]:0.276320,(Callicebus_donacophilus[&index=2]:0.156626,(Saimiri_sciureus[&index=20]:0.166654,(Aotus_trivirgatus[&index=1]:0.055799,Cebus_albifrons[&index=3]:0.121806)[&index=33]:0.060181)[&index=26]:0.002099)[&index=43]:0.198958)[&index=31]:0.109719,(Daubentonia_madagascariensis[&index=7]:0.204916,((((Galago_senegalensis[&index=8]:0.062634,Otolemur_crassicaudatus[&index=16]:0.083736)[&index=24]:0.129689,Perodicticus_potto[&index=18]:0.180867)[&index=29]:0.048537,(Nycticebus_coucang[&index=15]:0.191772,Loris_tardigradus[&index=12]:0.139268)[&index=37]:0.053495)[&index=41]:0.073039,((Microcebus_murinus[&index=14]:0.206292,Cheirogaleus_major[&index=4]:0.122795)[&index=39]:0.080094,((Lemur_catta[&index=10]:0.204318,Varecia_variegata_variegata[&index=22]:0.160840)[&index=40]:0.007776,(Lepilemur_hubbardorum[&index=11]:0.205821,Propithecus_coquereli[&index=19]:0.179995)[&index=38]:0.013664)[&index=27]:0.025400)[&index=35]:0.088251)[&index=28]:0.101114)[&index=34]:0.005263)[&index=36]:0.084039,Galeopterus_variegatus[&index=23]:0.217643)[&index=44];
