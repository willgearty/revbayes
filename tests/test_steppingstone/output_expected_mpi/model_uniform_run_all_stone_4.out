state	power	likelihood
4	0.07776	-21629.5
5	0.07776	-21647.8
6	0.07776	-21629.3
7	0.07776	-21636.8
8	0.07776	-21598.1
9	0.07776	-21605.5
10	0.07776	-21553.6
